app/javascript/packs/provider-dialogs-common.js

```diff
@@ -1,26 +1,127 @@
 import React from 'react';
-import renderModal from '../provider-dialogs/modal'
+import renderModal from '../provider-dialogs/modal';
 
-window.listenToRx(function(buttonData) {
-  if (!buttonData || buttonData.controller !== 'provider_dialogs') {
-    return;
-  }
+ManageIQ.angular.app.component('providerDialogUser', {
+  bindings: {
+    dialog: '<',
+    entityName: '@',
+    actionName: '@',
+  },
+  template: '<dialog-user dialog="$ctrl.dialog" refresh-field="$ctrl.refreshField(field)" on-update="$ctrl.setDialogData(data)"></dialog-user>',
+  controller: [function() {
+    let formData = {};
 
-  if (! buttonData.component_name) {
-    console.error('provider-dialogs: missing component_name', buttonData);
-    return;
-  }
+    /* this.refreshField = (field) => {
```

Comment:
refreshField is required by dialog-user..

not sure if it should return that do-nothing promise, or explicitly fail, but it should probably not fail on missing function..